cmd/zapi/main.go

```diff
@@ -3,25 +3,14 @@ package main
 import (
 	"fmt"
 	"os"
-
-	"github.com/brimdata/zed/cmd/zed/api"
-	_ "github.com/brimdata/zed/cmd/zed/api/auth"
-	_ "github.com/brimdata/zed/cmd/zed/api/drop"
-	_ "github.com/brimdata/zed/cmd/zed/api/get"
-	_ "github.com/brimdata/zed/cmd/zed/api/index"
-	_ "github.com/brimdata/zed/cmd/zed/api/info"
-	_ "github.com/brimdata/zed/cmd/zed/api/new"
-	_ "github.com/brimdata/zed/cmd/zed/api/post"
-	_ "github.com/brimdata/zed/cmd/zed/api/rename"
-	_ "github.com/brimdata/zed/cmd/zed/api/version"
-	"github.com/brimdata/zed/cmd/zed/root"
 )
 
 func main() {
-	root.Zed.Add(api.Cmd)
-	args := append([]string{"api"}, os.Args[1:]...)
-	if err := root.Zed.ExecRoot(args); err != nil {
-		fmt.Fprintf(os.Stderr, "%s\n", err)
-		os.Exit(1)
-	}
+	fmt.Fprintf(os.Stderr, "disabled for now")
```

Comment:
I happened to try installing this branch to see if it maybe fixed another issue that I was seeing, and I bumped into this:

```
$ zapi -h
disabled for now
```

Will help text be brought back before this gets merged?

Reply:
No, not until my follow up pr.